packages/destination-actions/src/destinations/launchdarkly-audiences/syncAudience/index.ts

```diff
@@ -0,0 +1,74 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { processPayload } from '../../launchdarkly-audiences/syncAudience/custom-audience-operations'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Sync Audience',
+  description: 'Sync Engage Audiences to LaunchDarkly segments',
+  defaultSubscription: 'type = "identify" or type = "track"',
+  fields: {
+    custom_audience_name: {
+      label: 'Custom Audience Name',
+      description: 'Name of custom audience list to which user identifier should added/removed',
+      type: 'string',
+      required: true,
+      default: {
+        '@path': '$.context.personas.computation_key'
+      }
+    },
+    segment_computation_action: {
```

Comment:
I don't think we need this here as it will allow customers to provide something arbitrary and may end up breaking it. I think we should be able to always the computation action will be the same.

Reply:
i agree that customers can easily break this however im not sure we have a work around for it. If we want to be able to access this property from the payload in our `perform` function then we need to define it as field. Segment doesnt send us the full/raw payload automatically in the perform function

Reply:
Gotcha - In that case I think there is a way we can make it hidden.

Reply:
I was looking for a [hidden property](https://github.com/segmentio/action-destinations#input-field-interface) but there wasnt one 😞

Reply:
i think there's a `type: 'hidden',` option for fields? example [here](https://github.com/launchdarkly/action-destinations/blob/1df145df02188923a70801bbba02fbfaed3b43f6/packages/destination-actions/src/destinations/linkedin-audiences/updateAudience/index.ts#L31)

Reply:
perfect! thank you